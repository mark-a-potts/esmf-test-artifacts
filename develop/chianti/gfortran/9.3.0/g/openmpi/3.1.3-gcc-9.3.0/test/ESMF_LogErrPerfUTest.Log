build time -- 2021-04-23 12:43:04.760193
20210423 124705.162 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124705.162 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124705.162 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124705.162 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124705.162 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124705.162 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124705.162 INFO             PET0 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124705.162 INFO             PET0 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124705.162 INFO             PET0 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124705.162 INFO             PET0 ESMF_COMM                   : openmpi
20210423 124705.162 INFO             PET0 ESMF_MOAB                   : enabled
20210423 124705.162 INFO             PET0 ESMF_LAPACK                 : enabled
20210423 124705.162 INFO             PET0 ESMF_NETCDF                 : enabled
20210423 124705.162 INFO             PET0 ESMF_PNETCDF                : disabled
20210423 124705.162 INFO             PET0 ESMF_PIO                    : enabled
20210423 124705.162 INFO             PET0 ESMF_YAMLCPP                : enabled
20210423 124705.162 INFO             PET0  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
20210423 124705.162 INFO             PET0  NUMBER_OF_PROCESSORS           4
20210423 124705.162 INFO             PET0  perfLogFoundError:         1000  iterations took   5.6249555200338364E-006  seconds. =>    5.6249555200338361E-009  per iteration.
20210423 124705.162 INFO             PET0  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
20210423 124705.162 INFO             PET0  perfLogFoundError:        10000  iterations took   5.4605072364211082E-005  seconds. =>    5.4605072364211083E-009  per iteration.
20210423 124705.162 INFO             PET0  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
20210423 124705.163 INFO             PET0  perfLogFoundError:       100000  iterations took   5.2623613737523556E-004  seconds. =>    5.2623613737523552E-009  per iteration.
20210423 124705.163 INFO             PET0  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
20210423 124705.168 INFO             PET0  perfLogFoundError:      1000000  iterations took   5.5145800579339266E-003  seconds. =>    5.5145800579339262E-009  per iteration.
20210423 124705.168 INFO             PET0  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
20210423 124705.168 INFO             PET0  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 99
20210423 124705.168 INFO             PET0  Ending Test, file ESMF_LogErrPerfUTest.F90, line 103
20210423 124705.168 INFO             PET0 Finalizing ESMF
20210423 124705.162 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124705.162 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124705.162 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124705.162 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124705.162 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124705.162 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124705.162 INFO             PET1 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124705.162 INFO             PET1 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124705.162 INFO             PET1 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124705.162 INFO             PET1 ESMF_COMM                   : openmpi
20210423 124705.162 INFO             PET1 ESMF_MOAB                   : enabled
20210423 124705.162 INFO             PET1 ESMF_LAPACK                 : enabled
20210423 124705.162 INFO             PET1 ESMF_NETCDF                 : enabled
20210423 124705.162 INFO             PET1 ESMF_PNETCDF                : disabled
20210423 124705.162 INFO             PET1 ESMF_PIO                    : enabled
20210423 124705.162 INFO             PET1 ESMF_YAMLCPP                : enabled
20210423 124705.162 INFO             PET1  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
20210423 124705.162 INFO             PET1  NUMBER_OF_PROCESSORS           4
20210423 124705.162 INFO             PET1  perfLogFoundError:         1000  iterations took   5.6067947298288345E-006  seconds. =>    5.6067947298288344E-009  per iteration.
20210423 124705.162 INFO             PET1  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
20210423 124705.162 INFO             PET1  perfLogFoundError:        10000  iterations took   5.4639065638184547E-005  seconds. =>    5.4639065638184551E-009  per iteration.
20210423 124705.162 INFO             PET1  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
20210423 124705.163 INFO             PET1  perfLogFoundError:       100000  iterations took   5.2614905871450901E-004  seconds. =>    5.2614905871450905E-009  per iteration.
20210423 124705.163 INFO             PET1  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
20210423 124705.168 INFO             PET1  perfLogFoundError:      1000000  iterations took   5.3324489854276180E-003  seconds. =>    5.3324489854276181E-009  per iteration.
20210423 124705.168 INFO             PET1  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
20210423 124705.168 INFO             PET1  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 99
20210423 124705.168 INFO             PET1  Ending Test, file ESMF_LogErrPerfUTest.F90, line 103
20210423 124705.168 INFO             PET1 Finalizing ESMF
20210423 124705.162 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124705.162 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124705.162 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124705.162 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124705.162 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124705.162 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124705.162 INFO             PET2 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124705.162 INFO             PET2 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124705.162 INFO             PET2 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124705.162 INFO             PET2 ESMF_COMM                   : openmpi
20210423 124705.162 INFO             PET2 ESMF_MOAB                   : enabled
20210423 124705.162 INFO             PET2 ESMF_LAPACK                 : enabled
20210423 124705.162 INFO             PET2 ESMF_NETCDF                 : enabled
20210423 124705.162 INFO             PET2 ESMF_PNETCDF                : disabled
20210423 124705.162 INFO             PET2 ESMF_PIO                    : enabled
20210423 124705.162 INFO             PET2 ESMF_YAMLCPP                : enabled
20210423 124705.162 INFO             PET2  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
20210423 124705.162 INFO             PET2  NUMBER_OF_PROCESSORS           4
20210423 124705.162 INFO             PET2  perfLogFoundError:         1000  iterations took   5.8249570429325104E-006  seconds. =>    5.8249570429325105E-009  per iteration.
20210423 124705.162 INFO             PET2  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
20210423 124705.162 INFO             PET2  perfLogFoundError:        10000  iterations took   5.2696093916893005E-005  seconds. =>    5.2696093916893004E-009  per iteration.
20210423 124705.162 INFO             PET2  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
20210423 124705.163 INFO             PET2  perfLogFoundError:       100000  iterations took   5.2616582252085209E-004  seconds. =>    5.2616582252085207E-009  per iteration.
20210423 124705.163 INFO             PET2  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
20210423 124705.168 INFO             PET2  perfLogFoundError:      1000000  iterations took   5.3683009464293718E-003  seconds. =>    5.3683009464293720E-009  per iteration.
20210423 124705.168 INFO             PET2  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
20210423 124705.168 INFO             PET2  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 99
20210423 124705.168 INFO             PET2  Ending Test, file ESMF_LogErrPerfUTest.F90, line 103
20210423 124705.168 INFO             PET2 Finalizing ESMF
20210423 124705.162 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124705.162 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 124705.162 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 124705.162 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 124705.162 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 124705.162 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 124705.162 INFO             PET3 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 124705.162 INFO             PET3 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 124705.162 INFO             PET3 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 124705.162 INFO             PET3 ESMF_COMM                   : openmpi
20210423 124705.162 INFO             PET3 ESMF_MOAB                   : enabled
20210423 124705.162 INFO             PET3 ESMF_LAPACK                 : enabled
20210423 124705.162 INFO             PET3 ESMF_NETCDF                 : enabled
20210423 124705.162 INFO             PET3 ESMF_PNETCDF                : disabled
20210423 124705.162 INFO             PET3 ESMF_PIO                    : enabled
20210423 124705.162 INFO             PET3 ESMF_YAMLCPP                : enabled
20210423 124705.162 INFO             PET3  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
20210423 124705.162 INFO             PET3  NUMBER_OF_PROCESSORS           4
20210423 124705.162 INFO             PET3  perfLogFoundError:         1000  iterations took   5.6368298828601837E-006  seconds. =>    5.6368298828601839E-009  per iteration.
20210423 124705.162 INFO             PET3  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
20210423 124705.162 INFO             PET3  perfLogFoundError:        10000  iterations took   5.2701216191053391E-005  seconds. =>    5.2701216191053394E-009  per iteration.
20210423 124705.162 INFO             PET3  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
20210423 124705.163 INFO             PET3  perfLogFoundError:       100000  iterations took   5.2629783749580383E-004  seconds. =>    5.2629783749580386E-009  per iteration.
20210423 124705.163 INFO             PET3  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
20210423 124705.168 INFO             PET3  perfLogFoundError:      1000000  iterations took   5.3890740964561701E-003  seconds. =>    5.3890740964561704E-009  per iteration.
20210423 124705.168 INFO             PET3  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
20210423 124705.168 INFO             PET3  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 99
20210423 124705.168 INFO             PET3  Ending Test, file ESMF_LogErrPerfUTest.F90, line 103
20210423 124705.168 INFO             PET3 Finalizing ESMF
