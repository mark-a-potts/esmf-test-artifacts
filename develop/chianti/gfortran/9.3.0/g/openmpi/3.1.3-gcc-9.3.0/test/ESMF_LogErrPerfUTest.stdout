build time -- 2021-04-23 12:43:04.760193
  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
  NUMBER_OF_PROCESSORS           4
  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 99
  Ending Test, file ESMF_LogErrPerfUTest.F90, line 103
  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 99
  Ending Test, file ESMF_LogErrPerfUTest.F90, line 103
  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 99
  Ending Test, file ESMF_LogErrPerfUTest.F90, line 103
  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 99
  Ending Test, file ESMF_LogErrPerfUTest.F90, line 103
  PET           2  Test Elapsed Time    7.1949999999999985       msec.
  PET           1  Test Elapsed Time    7.0819999999999999       msec.
  PET           0  Test Elapsed Time    7.3060000000000000       msec.
  PET           3  Test Elapsed Time    7.1570000000000018       msec.
